client/extensions/wp-super-cache/fix-config.jsx

```diff
@@ -2,36 +2,28 @@
  * External dependencies
  */
 import React from 'react';
+import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
 import Button from 'components/button';
 import Card from 'components/card';
 import SectionHeader from 'components/section-header';
-import WrapSettingsForm from './wrap-settings-form';
 
-const FixConfig = ( { translate } ) => {
-	return (
-		<div>
-			<SectionHeader label={ translate( 'Fix Configuration' ) } />
-			<Card>
-				<form>
-					<div>
-						<Button
-							compact
-							type="submit">
-							{ translate( 'Restore Default Configuration' ) }
-						</Button>
-					</div>
-				</form>
-			</Card>
-		</div>
-	);
-};
+const FixConfig = ( { translate } ) =>
+	<div>
+		<SectionHeader label={ translate( 'Fix Configuration' ) } />
+		<Card>
+			<form>
+				<div>
+					<Button
+						compact>
```

Comment:
Oh and BTW, do we need all those levels of wrapping? The `<form>` and `<div>` specifically...